### ISSUE TITLE
Proposal: remove out.gif when complete

### ISSUE THREAD

**Issue by jmooring** [charmbracelet/vhs#168]
With:

```
# Output
Output demonstration.mp4
Output demonstration.webm
```

The MP4 and WebM files are created as expected, but there's an extraneous "out.gif" file that's left over. It would be nice to remove that when finished.

**Comment by maaslalani** [comment]
Totally, this is a good point. In fact, we should simply not generate the `out.gif` in the first place (as it is not necessary to produce the mp4/webm outputs, they are generated with the frame sequences)

**Comment by rm-dr** [comment]
Seems the error is in the default value of `VideoOptions.Output` in `video.go`. It's always set to `out.gif`!

A quick fix is:

1. Remove the default GIF output in `video.go` (line 71)
2. Add an if like the following inside `MakeGIF`:

```go
func MakeGIF(opts VideoOptions) *exec.Cmd {
	if opts.Output.GIF == "" &&
		opts.Output.WebM == "" &&
		opts.Output.MP4 == "" {

		opts.Output.GIF = "out.gif"
	}

	// The rest of the function...
```


This feels like a hack, though. Is there a better way?

**Comment by maaslalani** [comment]
Fixed by #172
